AsyncDisplayKit/ASEditableTextNode.mm

```diff
@@ -133,7 +135,7 @@ - (void)didLoad
   [self.view addSubview:_placeholderTextKitComponents.textView];
 
   // Create and configure our text view.
-  _textKitComponents.textView = [[_ASDisabledPanUITextView alloc] initWithFrame:CGRectZero textContainer:_textKitComponents.textContainer];
+  _textKitComponents.textView = _textView;
```

Comment:
Let's refactor this area to do something like this:

UITextView *textView = self.textView; // The textView accessor method should set up _textKitComponents.textView

Then just use textView directly in the following six lines.  This eliminates 6 message sends as well as makes the code more readable.